src/pages/SignUp/signUp.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Link } from "react-router-dom";
+
+import { Header, Footer, Input } from '../../components';

```

Comment:
use absolute path instead of relative path.

refer link : https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i

Reply:
Thank you will keep this in mind moving forward.